packages/webapp/pages/onboarding.tsx

```diff
@@ -309,14 +330,21 @@ export function OnboardPage(): ReactElement {
         showOnboardingPage={showOnboardingPage}
         setAuth={setAuth}
         customActionName={customActionName}
-        onClickCreateFeed={onClickCreateFeed}
+        onClick={
+          activeScreen === OnboardingStep.AndroidApp
+            ? onCompleteOnboarding
```

Comment:
We should never call this directly I think, it now misses the settings set, right?

Reply:
@rebelchris We're not calling it differently than we used to, just checking which function to pass?

This mutation is still getting called if that's what you're referring to:

![Screenshot 2024-11-28 at 11 31 20](https://github.com/user-attachments/assets/0762fb19-8ce0-4342-b5fa-e06763e39b76)

Reply:
How though?
You either call `onCompleteOnboarding` or `onClickCreateFeed` now?
I'm not sure why it needs to be something different either way.

It's common practice already we have steps you can optionally skip it shouldn't need new logic right?

Reply:
@rebelchris Sorry, I might've misunderstood what exactly you were referring to. I thought you were talking about how we pass the function to the component, not that you meant we shouldn't call `onCompleteOnboarding` itself explicitly.

I primarily did this to not re-call the setting of the settings (what a sentence 😅) again, but if you don't think it's a big deal I'll revert this!